source/nodejs/adaptivecards-designer/src/dialog.ts

```diff
@@ -168,7 +168,7 @@ export abstract class Dialog {
                 this.onClose(this);
             }
             // Removing the focus action on close, it has been handled via onClose method of Dialog consume place.
-            // this._originalFocusedElement.focus();
+            this._originalFocusedElement.focus();
```

Comment:
Does anyone know why this line was commented out in the first place? I am having trouble understanding the comment above it

Reply:
@matthidinger  Look like it's introduced by [PR 4893](https://github.com/microsoft/AdaptiveCards/pull/4893/files#diff-6312c8913516e0c6baf4e4068c47bff7dc35c113cd5a9821e330ace260081a8e)

Reply:
I think it's fine to reintroduce this